src/applications/personalization/view-dependents/components/ViewDependentsList/ViewDependentsListItem.jsx

```diff
@@ -20,6 +28,17 @@ function ViewDependentsListItem(props) {
     stateKey,
   } = props;
 
+  const handleClick = () => {
+    setOpen(prevState => !prevState);
+    if (open) {
+      const focusEl = document?.querySelectorAll('.mng-dependents-name')?.[
```

Comment:
Do we want the name set as the focus element or the first form field?

Reply:
It's a little weird, but this activates when the form closes, so having focus on the name makes a little more sense to me.

Reply:
gotcha